src/main/java/com/epam/task/third/data/DataReader.java

```diff
@@ -0,0 +1,38 @@
+package com.epam.task.third.data;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class DataReader {
+    private String inputFileName;
+
+    private static final Logger LOGGER = Logger.getLogger(String.valueOf(DataReader.class));
+
+    public DataReader() {}
+
+    public List<String> readFile(String inputFileName) throws DataException {
+        List<String> stringList = new ArrayList<>();
+        try {
+        BufferedReader reader = new BufferedReader(new FileReader(inputFileName));
+            String line;
+            while ((line = reader.readLine()) != null) {
+                stringList.add(line);
+            }
+        }
+        catch (IOException e) {
+            LOGGER.log(Level.SEVERE, e.getMessage(), e);
```

Comment:
LOGGER.error(...
where you were in our classes when we discussed all this?
do not log if you re-throw exception